packages/terra-form-input/src/Input.module.scss

```diff
@@ -27,6 +27,14 @@
       font-style: var(--terra-form-input-placeholder-font-style);
     }
 
+    /* TODO: Remove vendor prefix once we update to autoprefixer v9 */
+    /* stylelint-disable selector-no-vendor-prefix */
+    &:-ms-input-placeholder {
+      color: var(--terra-form-input-placeholder-color, #70787c);
+      font-style: var(--terra-form-input-placeholder-font-style);
+    }
+    /* End of vendor prefix */
```

Comment:
want to re-enable the lint rule here?

Reply:
added : https://github.com/cerner/terra-core/blob/88ec784c6421f99916bf8abcc9723556f18b0bbb/packages/terra-form-input/src/Input.module.scss#L37